jena-arq/src/main/java/org/apache/jena/sparql/engine/iterator/QueryIterSort.java

```diff
@@ -89,6 +92,7 @@ public SortedBindingIterator(final QueryIterator qIter) {
             // iterator in a try/finally block, and thus will call
             // close() themselves.
             catch (QueryCancelledException e) {
+		QueryIterSort.this.close();
```

Comment:
There are tabs in various plaxces.  Could you follow the convention of using spaces? Display works better. Your tab size != all other tab sizes != Github's handling of tabs.